jobs/record_balance_history_test.go

```diff
@@ -193,7 +193,7 @@ func TestRecordBalanceHistoryTimestampTruncation(t *testing.T) {
 }
 
 func TestRecordBalanceHistoryConflictHandling(t *testing.T) {
-	pool := database.CreateTestDatabase(t, "test_api")
+	pool := database.CreateTestDatabase(t, "test_jobs")
```

Comment:
fyi @dylanjeffers sorry I didn't catch this in initial review - each "package" of tests in go runs in parallel, and only one database can be created from a template at a time, so each package has its own template to use for tests to avoid race conditions

super annoying footgun but it makes tests faasstt